### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by SouravApiDev** [dsdanielpark/Bard-API#220] [General]
Exception                                 Traceback (most recent call last)
[<ipython-input-3-9bb8229ac408>](https://localhost:8080/#) in <cell line: 5>()
      3 os.environ['_BARD_API_KEY']="epiIk6A2qHAOPEfJcjRfpd6L-8ZZVMQOpkSK6HrN-J1qASFreTknUC2GIh3QC8LWQpHpwg."
      4 
----> 5 Bard().get_answer("what is pi?")['content']

**Comment by dsdanielpark** [comment]
First, please check if it runs correctly on a web UI in the browser.
Turn off all browsers, log out, and then log in to refresh the cookie values.
The most likely scenario is that the cookie values have been updated internally. Additionally, please review the links attached to the issue page for further details. Thank you.

```python
from bardapi import Bard
import os

token="epiIk6A2qHAOPEfJcjRfpd6L-8ZZVMQOpkSK6HrN-J1qASFreTknUC2GIh3QC8LWQpHpwg."

Bard(token=token).get_answer("what is pi?")['content']
```
